src/app/home/home.css

```diff
@@ -0,0 +1,131 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+#landingPage {
+    height: 85%;
+    width: auto;
+    background-image: url('../../img/landingPageImage.jpeg');
+    background-size: cover;
+    position: relative;
+    background-position: center center;
+    color: white;
+}
+
```

Comment:
Why is this file not in the style folder in /src

Reply:
moving it asap